rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -272,11 +274,19 @@ def handle_compression_offers(offers):
     uri = '{}://{}:{}'.format(protocol, address, port)
     factory = WebSocketServerFactory(uri)
     factory.protocol = RosbridgeWebSocket
-    factory.setProtocolOptions(
-        perMessageCompressionAccept=handle_compression_offers,
-        autoPingInterval=ping_interval,
-        autoPingTimeout=ping_timeout,
-    )
+    if autobahn.__version__ == '0.10.3':
```

Comment:
Comparing to a specific version is very brittle and not likely to work for users of other OS distros.  Since the distutils package is expected to be available in ROS distributions and we know exactly which version adds the feature, please use [`distutils.version.LooseVersion`](http://epydoc.sourceforge.net/stdlib/distutils.version.LooseVersion-class.html).  e.g.
```python
from distutils.version import LooseVersion

...

if LooseVersion(autobahn.__version__) < LooseVersion("0.15.0"):
    ...
```

Reply:
good idea, if you know a better way and are a maintainer can you fix this bug? Otherwise I will submit another fix when I can.